src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -1096,6 +1100,15 @@ function transitionGeneticTrack(
         return;
     } else if (nextSpec && !prevSpec) {
         // Add track
+        const tooltipFn =
+            typeof createCustomOncoprintTooltip !== 'undefined'
```

Comment:
in appBootstrap, lets make an global object
 ```
getBrowserWindow().customJSFunctions = {};
```   
we can then type the potential functions (e.g. ```createCustomOncoprintTooltip```   the customJs files we load can assign to that object, making it clear where they come from.

Reply:
Changed in [a3ce63d](https://github.com/cBioPortal/cbioportal-frontend/pull/4677/commits/a3ce63d1593bdacaaab14103fd9970c44cf9b426), also updated example in PR instructions